notebooks/ch-algorithms/superdense-coding.ipynb

```diff
@@ -2484,7 +2484,7 @@
    "provenance": []
   },
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "Python 3",
```

Comment:
I believe this change isn't related to https://github.com/Qiskit/platypus/issues/715.